libs/elements/cards/src/lib/elewa-project-item-card/elewa-project-item-card.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ProjectItem } from '@elewa-website/models/sections/projects/cards'
+
+@Component({
+  selector: 'elewa-website-elewa-project-item-card',
+  templateUrl: './elewa-project-item-card.component.html',
+  styleUrls: ['./elewa-project-item-card.component.scss'],
+})
+export class ElewaProjectItemCardComponent implements OnInit {
+  
+  // Import of the interface
```

Comment:
No need for this comment. Only use JsDoc comments to describe properties and methods. This allows us to get some good Intellisense.